src/Platform/Microsoft.Testing.Platform/UI/NonAnsiTerminal.cs

```diff
@@ -0,0 +1,240 @@
+    {
+        Append(path);
+        if (lineNumber.HasValue)
+        {
+            Append($":{lineNumber}");
+        }
+    }
+
+    public void SetColor(TerminalColor color)
+    {
+        if (_isBatching)
+        {
+            _console.Write(_stringBuilder.ToString());
+            _stringBuilder.Clear();
+        }
+
+        _console.SetForegroundColor(ToConsoleColor(color));
+    }
+
+    public void ResetColor()
+    {
+        if (_isBatching)
+        {
+            _console.Write(_stringBuilder.ToString());
+            _stringBuilder.Clear();
+        }
+
+        _console.SetForegroundColor(_defaultForegroundColor);
+    }
+
+    public void ShowCursor()
+    {
+        // nop
+    }
+
+    public void HideCursor()
+    {
+        // nop
+    }
+
+    public void StartUpdate()
+    {
+        if (_isBatching)
+        {
+            throw new InvalidOperationException("Console is already in batching mode.");
+        }
+
+        _stringBuilder.Clear();
+        _isBatching = true;
+    }
+
+    public void StopUpdate()
+    {
+        _console.Write(_stringBuilder.ToString());
+        _isBatching = false;
+    }
+
+    private ConsoleColor ToConsoleColor(TerminalColor color) => color switch
+    {
+        TerminalColor.Black => ConsoleColor.Black,
+        TerminalColor.DarkRed => ConsoleColor.DarkRed,
+        TerminalColor.DarkGreen => ConsoleColor.DarkGreen,
+        TerminalColor.DarkYellow => ConsoleColor.DarkYellow,
+        TerminalColor.DarkBlue => ConsoleColor.DarkBlue,
+        TerminalColor.DarkMagenta => ConsoleColor.DarkMagenta,
+        TerminalColor.DarkCyan => ConsoleColor.DarkCyan,
+        TerminalColor.DarkWhite => ConsoleColor.White,
+        TerminalColor.Default => _defaultForegroundColor,
+        TerminalColor.Gray => ConsoleColor.Gray,
+        TerminalColor.Red => ConsoleColor.Red,
+        TerminalColor.Green => ConsoleColor.Green,
+        TerminalColor.Yellow => ConsoleColor.Yellow,
+        TerminalColor.Blue => ConsoleColor.Blue,
+        TerminalColor.Magenta => ConsoleColor.Magenta,
+        TerminalColor.Cyan => ConsoleColor.Cyan,
+        TerminalColor.White => ConsoleColor.White,
+        _ => _defaultForegroundColor,
+    };
+
+    public void EraseProgress()
+    {
+        // nop
+    }
+
+    public void RenderProgress(TestWorker?[] progress)
+    {
+        StartUpdate();
+        try
+        {
+            foreach (TestWorker? p in progress)
+            {
+                if (p == null)
+                {
+                    continue;
+                }
+
+                string durationString = $" ({p.Stopwatch.Elapsed.TotalSeconds:F1}s)";
+
+                int passed = p.Passed;
+                int failed = p.Failed;
+                int skipped = p.Skipped;
+
+                string? detail = !RoslynString.IsNullOrWhiteSpace(p.Detail) ? $"- {p.Detail}" : null;
+                Append('[');
+                SetColor(TerminalColor.DarkGreen);
+                Append("✔️");
```

Comment:
Could we have a green mark?

Reply:
✅ yes, this with a box. This one is more consistently green across systems, but also the only one that is outlined. 

https://emojipedia.org/check-mark-button#designs

Reply:
replaced.